app/controllers/authentication_controller.rb

```diff
@@ -0,0 +1,17 @@
+class AuthenticationController < ApplicationController
```

Comment:
@vamshikrishnaprasthana why this old file is still present, after renaming to authentications_controller